andela/trime
# Replace tracking status with a timer

#### What does this PR do?
Replaces the status text on the `TrackingWidget` with a running timer.

#### How should this be manually tested?
Go to project page and start tracking time using the tracking widget. When you start, you'll see the amount of time you've tracked increasing every second. If you pause, it stops increasing. If you resume, it starts increasing every second again.

#### What are the relevant pivotal tracker stories?
[ Replace the status text on the `TrackingWidget` with a running timer.](https://www.pivotaltracker.com/story/show/120097169)
